lib/file-utilities.js

```diff
@@ -169,20 +157,25 @@ function streamAllLogs(config, callback) {
   async.each(config.logdir, (dir, done) => {
     getFiles(config, dir, (err, logfiles) => {
       if (!err && logfiles.length > 0) {
+        conf = config
```

Comment:
Ugh, I hate that I never got to finish this.  `conf = config` is terrible, but oh well.

Reply:
Yeah I had my issues with that too. I just left it as is for the sake of not messing with this too much. The thought was this will eventually be deprecated.